lmdeploy/turbomind/turbomind.py

```diff
@@ -87,6 +88,37 @@ def _update_tm_config(dst: TurbomindModelConfig, src: TurbomindEngineConfig):
     return TurbomindModelConfig.from_dict(dst_dict)
 
 
+def _compare_individual_gpu_memory():
+    logging.basicConfig(level=logging.INFO)
+
+    devices = os.getenv('CUDA_VISIBLE_DEVICES', '')
```

Comment:
Getting device list from `CUDA_VISIBLE_DEVICES` is not necessary. 

Devices the process can see is already filtered and rearranged according to `CUDA_VISIBLE_DEVICES` by the driver.